data/successors/successor 1 prologue.txt

```diff
@@ -927,7 +927,7 @@ mission "Successors: Trusted"
 			`	"Captain! Well-met again! Thou hast arrived at a fortunate hour, for there's still some minutes 'fore this day's meetings are scheduled to begin."`
 			label questions
 			choice
-				`	"We'd better get started then.`
+				`	"We'd better get started then."`
```

Comment:
Pretty sure it should actually be this:

```suggestion
				`	"We'd better get started, then."`
```

But it's still a good fix either way